src/elements/content-sidebar/activity-feed/annotations/__tests__/AnnotationActivity.test.js

```diff
@@ -54,22 +54,35 @@ describe('elements/content-sidebar/ActivityFeed/annotations/AnnotationActivity',
         CommentForm.default = jest.fn().mockReturnValue(<div />);
     });
 
-    test('should correctly render annotation activity', () => {
-        const unixTime = new Date(TIME_STRING_SEPT_27_2017).getTime();
-        const item = {
-            ...mockAnnotation,
-            permissions: { can_delete: true },
-        };
-
-        const wrapper = getWrapper({ item });
-        const activityLink = wrapper.find(AnnotationActivityLink);
-
-        expect(wrapper.find('ActivityTimestamp').prop('date')).toEqual(unixTime);
-        expect(activityLink.prop('message')).toEqual({ ...messages.annotationActivityPageItem, values: { number: 1 } });
-        expect(activityLink.props()).toMatchObject(generateReginTags());
-        expect(wrapper.find('AnnotationActivityMenu').length).toEqual(1);
-        expect(wrapper.find('ActivityMessage').prop('tagged_message')).toEqual(mockActivity.item.description.message);
-    });
+    test.each`
+        canDelete | canEdit
```

Comment:
Should we have a separate test for `${false} | ${false}`?

Reply:
Yes we should.